gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -1446,6 +1446,21 @@ protected void configureConversionUtilities(Project project, SourceSet sourceSet
         project.getLogger().lifecycle("You can use '-PconvertToPdl.reverse=true|false' to change the direction of conversion.");
       }));
     });
+
+    project.getTasks().create(sourceSet.getTaskName("reformat", "Pdl"), TranslateSchemasTask.class, task ->
```

Comment:
should we make this optional to run? or we run this default whenever we make a change to the format if this is the case are you planning to remove it later?

Reply:
This task is already optional, it is not linked to any build task.
It can be run by running `ligradle reformatPdl`, we don't have to remove it later. One way this can be used is to auto format PDL files. MP owners can use this to keep the formatting consistent.